### DISCUSSION TITLE
Setzen des Limit Wertes über iobroker/MQTT

### DISCUSSION THREAD

**Discussion by frontlinerde** [tbnobody/OpenDTU#239] [Q&A]
Hi, gibts ne Möglichkeit den Limitwert über MQTT zu setzen?
Wenn ich ihn aktuell über den iobroker setze, springt er auf den in der OpenDTU-Webseite eingestellten Wert zurück.

Geht das aktuell überhaupt?

**Comment by johannesosterberg** [comment]
Ja, das geht hervorragend. Du musst nur ein MQTT Nachricht publishen z. B. mit dem MQTT-Explorer.

`solar/Seriennummer_deines_Inverters/cmd/limit_nonpersistent_absolute `

Und dann einen RAW-Wert, hier in Watt übertragen.

Das kann man natürlich auch mit NODE-Red oder ähnlichem machen.

**Reply by tbnobody** [reply]
Eine genaue Doku über die zur verfügung stehenden Topics ist in der README verlinkt bzw. hier zu finden: https://github.com/tbnobody/OpenDTU/blob/master/docs/MQTT_Topics.md

**Comment by frontlinerde** [comment]
der Zweig "[serial]/cmd" exisitiert bei mir erst garnicht
Da gibt es nur 

[serial]/0
[serial]/1
[serial]/2
[serial]/device
[serial]/settings
[serial]/status

iobroker mit mqtt-Adapter

**Reply by kpwg** [reply]
Den Zweig musst Du für die Übergabe der Daten/Parameter selbst anlegen.

btw: mit FHEM klappt's auch ;)

**Reply by frontlinerde** [reply]
hab ich parallel dazu auch gemacht und es scheint zu gehen.
Wobei ich ihn beim anlegen von [serial]/cmd/power erstmal ausgeschalten habe
